Makefile

```diff
@@ -39,12 +39,16 @@ ARFLAGS = rcs
 LDFLAGS = -pthread
 LDLIBS = $(TEST_LIBS)
 
+LN = ln
 CP = cp
 MKDIR = mkdir
 SUDO =
 
 CCTZ = cctz
 CCTZ_LIB = lib$(CCTZ).a
+CCTZ_SHARED_LIB = lib$(CCTZ).so.2.0
```

Comment:
Perhaps the GNU dependency is already there, with VPATH and -include, but perhaps it is still clearer this way.